calcom/cal.com
# fix: Remove Initial Credential Parsing For GCal

## What does this PR do?



This PR removes the initial Zod parsing of GCal credentials. This was causing problems for credentials created by 3rd party platforms. 

Fixes # (issue)



## Requirement/Documentation



- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change





## How should this be tested?



- Setup credential syncing by filling in the correct .env variables and setting a webhook that returns the access token for GCal
- Connect GCal to get the access token for the webhook
- Force a refresh of the token to call the webhook
- Calendars should be listed correctly
- When creating a new booking, the event should be created on the calendar

## Mandatory Tasks


## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
